Header header

# Steering Wheel
float32 steering_wheel_angle # deg
float32 steering_wheel_speed # ??
uint8 steering_stat
uint8 msg_count
uint8 checksum
